optimizer-express/demo/lib/GoogleFontsPreconnectTransformer.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/**
+ * Adds a preconnect instruction to `fonts.gstatic.com` when the Google Fonts CSS
+ * is used in the page.
+ *
+ * The Google Fonts CSS requests fonts `https://fonts.gstatic.com` after it is loaded.
+ * Browsers only connect to this domain after the CSS has been downloaded and the CSSOM has been
+ * constructed.
+ *
+ * By preconnecting earlier, the DNS resolution, TCP connection and SSL handshake are performed
+ * sooner, saving up to 1 second on regular 3G networks, resulting on the Fonts being loaded
+ * earlier.
+ *
+ * The transformer will only issue the preconnect instruction if Google Fonts is used on the page.
+ */
+class GoogleFontsPreconnectTransformer {
```

Comment:
Can you move this to the main transformer dir.

Reply:
Done. Added tests too.